src/database/queries.rs

```diff
@@ -71,3 +73,15 @@ where
         .filter(transactions::id.eq(tx_id))
         .first::<Transaction>(&conn)
 }
+
+pub async fn delete_txs<Context>(ctx: &Context, current_epoch: u128, epoch_amount: u128) -> Result<usize, Error>
+where
+    Context: QueryContext,
+{
+    let epochs : Vec<Epoch> = (0..epoch_amount).map(|i| Epoch(current_epoch - i)).collect();
+    let conn = ctx.get_db_connection();
+    let txs = transactions
+        .filter(transactions::epoch.ne_all(epochs));
```

Comment:
This filter is not enough, we should not delete transactions that are not confirmed. Additionally we should have a comment here that transactions that caused slashing can be deleted after some longer period, but we don't have that state for transactions at this point so this functionality needs to be added later.